resources/js/frontend.js

```
@@ -5,43 +5,44 @@
 
-function scrollTobottom() {
+function scrollToBottom() {
+    let mainChatInbox = $('.wsus__chat_area_body');
     mainChatInbox.scrollTop(mainChatInbox.prop("scrollHeight"));
 }
 
-window.Echo.private('message.'+ USER.id).listen(
-    "MessageEvent",
-    (e) => {
+if (typeof USER !== 'undefined') {
+    window.Echo.private('message.' + USER.id).listen('MessageEvent', (e) => {
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)